Adaptors/Memory/src/ArmoniK.Core.Adapters.Memory.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <IsPackable>true</IsPackable>
+    <Nullable>enable</Nullable>
```

Comment:
Why is this here ? It's reformatting pr !

Reply:
Because it forces resharper to use `?` instead of `[CanBeNull]`.